R/plot3D.x3p.file.R

```diff
@@ -70,7 +76,7 @@ plot3D.x3p.file<-function(x3p.surf.file.info, num.x.pts=NULL, num.slices=NULL, a
   xinc<-as.numeric(head.info["x.inc"]) #should be microns already
   xaxis<-seq(from=0,to=xinc*(num.pts.per.line-1),length.out=num.x.pts)
   dec.xaxis<-xaxis[dec.col.idxs]
-#   
+  #   
   num.lines<-as.numeric(head.info["num.lines"])
   yinc<-as.numeric(head.info["y.inc"]) #should be microns already FIX THIS LATER
```

Comment:
This is where the error is. If num.slices is NULL, seq stops. Changed num.slices to length(dec.row.idxs).